### ISSUE TITLE
You cannot arow up/down through the cards, you have to use tab.

### ISSUE THREAD

**Issue by msftedad** [microsoft/vsts-extension-retrospectives#1200]
#A11y #accessibility

**Describe the bug**
You cannot arow up/down through the cards, you have to use tab. Users may find it challenging to navigate through the cards as you can't access cards with up/down arrow keys. screen reader users will find it difficult to access cards.

**Are you using the Retrospectives Tool on-prem?**
No

**To Reproduce**

1. open the URL https://dev.azure.com/eDADPlayground/SCRUMProject .
2. Expand left navigation and select boards-->Retrospectives.
3. Now navigate to cards using up/down arrow keys.
4. Verify whether cards are accessible with up/down arrow keys or not.

**Expected behavior**
You should be able to use arow up/down through the cards.

**Screenshots**

https://github.com/user-attachments/assets/5af3ebf6-5dea-400b-8093-cd1ead2017be

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/04973ea7-70ca-44f8-a5d6-53f3ca3042b8" />

**What screen type are you seeing this on ?**
Desktop Windows 11

**Additional context**
